packages/nuxt/src/app/composables/asyncData.ts

```diff
@@ -146,14 +149,15 @@ export function useAsyncData<
 
   options.lazy = options.lazy ?? false
   options.immediate = options.immediate ?? true
+  options.deep = options.deep ?? asyncDataDefaults.deep
```

Comment:
If deep is expected to be default behavior, can't we simply chain with `?? true` here to avoid inlining defaults?

Reply:
I'm not sure that `{ deep: true }` is heavier than inlining `{}` and `asyncDataDefaults.deep ?? true`. Or am I missing something?

Reply:
Mainly consistency and readability (noticed lazy/immediate defaults are already inlined now...)